.github/ISSUE_TEMPLATE/native_bug_report.yml

```diff
@@ -0,0 +1,77 @@
+name: Bug Report for Quarkus native executables
```

Comment:
Could you check in a personal project if we can order templates by changing the name of the YAML file? Because if this gets not just below Bug report, I think people won't see it at all.

If adjusting the YAML file name with `1_...`, `2_...` works, we should probably do that.

Reply:
Good catch, I kept the name start to be "Bug Report" to preserve alphabetical order, but the order is based on the yaml file name not the name template name.

Renaming the yaml file seems to do the trick:

![image](https://github.com/quarkusio/quarkus/assets/1435395/04ae2572-e5d3-45af-ab76-4b184fafe9b0)